run.py

```diff
@@ -112,12 +112,10 @@ def main(args):
                 )
 
                 feature_names = model.get_human_readable_feature_names()
-                for i, (importance, index, is_positive) in enumerate(
-                    importance["importances"]
-                ):
-                    print(
-                        f'{i + 1}. \'{feature_names[int(index)]}\' ({"+" if (is_positive) else "-"}{importance})'
-                    )
+
+                model.print_feature_importances(
+                    importance["importances"], feature_names, predicted_class=probas
```

Comment:
The name of the parameter should change too, since it's no longer `predicted_class`